robot/basestation/static/js/pages/science.js

```diff
@@ -0,0 +1,352 @@
+// turn table graphics
+const twopi = 2 * Math.PI
+const pi = Math.PI
+
```

Comment:
Another thing, the callback timeout doesn't seem to work on the science page. You can test it by trying to activate the listener after running the task handler locally with `rosrun task_handler task_handler_server.py`